yellowbrick/model_selection/importances.py

```diff
@@ -218,13 +224,24 @@ def fit(self, X, y=None, **kwargs):
         else:
             self.features_ = np.array(self.labels)
 
+        if self.top_n and self.top_n > self.features_.shape[0]:
+            raise YellowbrickValueError(
+                "top_n '{}' cannot be greater than the number of "
+                "features '{}'".format(self.top_n, self.features_.shape[0])
+            )
+
         # Sort the features and their importances
         if self.stack:
             sort_idx = np.argsort(np.mean(self.feature_importances_, 0))
             self.features_ = self.features_[sort_idx]
             self.feature_importances_ = self.feature_importances_[:, sort_idx]
         else:
             sort_idx = np.argsort(self.feature_importances_)
+            if self.top_n:  # Keep only the top or bottom n examples
+                if self.top_n > 0:
+                    sort_idx = sort_idx[-self.top_n:]
+                else:
+                    sort_idx = sort_idx[:-self.top_n]
```

Comment:
I totally agree with the worry of confusion. I had hoped that the idea of using a negative number to grab from the back instead of the front would be simple since we see it all the time in python slicing. I'll ponder on some better nomenclature, and hope that the solution is not to have a separate `bottom_n` parameter entirely.

Speaking of nomenclature, would you prefer `top_n` or `topn`?

Reply:
Slicing is a very good point and also an excellent analogy. If we can't come up with a better name, I think a negative `top_n` is preferable to also having a `bottom_n` parameter. 

As for names, I do prefer `topn` slightly over `top_n` - however, I think this is a good place for you to make your mark on Yellowbrick, so please feel free to select your preference (let me know, and I'll be careful to use that arg from here on out).

Reply:
I default to snake case, but will alter to `topn` to be consistent with `xlabel` also in `FeatureImportances`